runtime/python/pose_estimation/pose_estimation_utils.py

```diff
@@ -0,0 +1,506 @@
+from pathlib import Path
+import numpy as np
+import cv2
+from PIL import Image
+from hailo_platform import HEF
+from zenlog import logging
```

Comment:
use same logging for the entire example

Reply:
this import is done in order to interpert the data type in: 
`def log_layer_info(
    layers: list, log: logging.Logger, layer_type: str = 'Input'
) -> None:`

So, in the meanwhile didn't change anything yet.

Reply:
removed the function.
the import is important for the logging in check_process_errors.
i can move check_process_errors (and perhaps output_data_type2dict as well) to pose.estimation.py